plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -423,6 +439,11 @@ def set_pre_stop
       end
     end
 
+    def init_containers
```

Comment:
I think we have to delete them from annotations and spec to avoid errors from unexpected keys being used

Reply:
You mean something like what I'm doing [here](https://github.com/zendesk/samson/blob/1469bbbd9281b713955e8155352076648b21c243/plugins/kubernetes/app/models/kubernetes/template_filler.rb#L246) and [here](https://github.com/zendesk/samson/blob/1469bbbd9281b713955e8155352076648b21c243/plugins/kubernetes/app/models/kubernetes/template_filler.rb#L249)?